spec/app/models/metasploit_data_models/module_run_spec.rb

```diff
@@ -0,0 +1,131 @@
+require 'spec_helper'
+
+describe MetasploitDataModels::ModuleRun do
+
+  subject(:module_run){FactoryGirl.build(:metasploit_data_models_module_run)}
+
+  context "database columns" do
+    it { is_expected.to have_db_column(:attempted_at).of_type(:datetime) }
+    it { is_expected.to have_db_column(:fail_detail).of_type(:text) }
+    it { is_expected.to have_db_column(:fail_reason).of_type(:string) }
+    it { is_expected.to have_db_column(:module_detail_id).of_type(:integer) }
```

Comment:
Mdm::Module::Detail are wiped on each restart of prosvc or msfconsole, so this foreign key shouldn't even exist and only module_full_name should be used.